src/drivers/char/terminal/vtparse.c

```diff
@@ -46,10 +46,11 @@ static void do_action(struct vtparse *parser, vt_action_t action, char ch) {
 		break;
 
 	case VT_ACTION_COLLECT:
-		if (token->attrs_len == -1) {
```

Comment:
No, I mean that this change (remove checking buffer overflow) can influence to another templates.
So, why is this required?